examples/resources/packageinstall/resource.tf

```diff
@@ -21,6 +21,8 @@ resource "tanzu-mission-control_package_install" "create_package_install" {
       }
     }
 
-    inline_values = { "test" : "test" }
+    path_to_inline_values = "<inline-values-file-path>"
```

Comment:
can you please provide the concrete example with the file as a sample?

Reply:
Sure.

Reply:
Updated.